### PR TITLE
feat: application table search + UI

### PR DESCRIPTION
**Pull request by zaki-amin**
Several UI improvements for the application table
Screenshot 2024-10-10 at 17 41 34
- Search bar in table toolbar
- Row count with pagination controls
- Alternating row colours (grey <-> white) for improved readability
- Enums rendered in user-friendly way
- WP column shortened and coloured
- Column separators and table border
- Forms grouped into one column

### CODE REVIEW

File: lib/utils.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { NextAction } from '@prisma/client'
 import { Decimal } from '@prisma/client/runtime/binary'
+import _ from 'lodash'
```
Comment (procaconsul) [inline lib/utils.ts:3]:
```suggestion
import capitalize from 'lodash'
```

File: lib/utils.ts [hunk 2]
```diff
@@ -20,3 +21,20 @@ const orderedActions: NextAction[] = Object.values(NextAction)
...
 }
+
+// convert to title case and replace '_ 'with ' '
+export function prettifyOption(value: string): string {
+  // leave emails untouched
+  if (value.includes('@')) return value
+  const ACRONYMS = ['Tmua', 'Ug', 'Gcse', 'Xii']
+  return value
+    .split('_')
+    .map(_.capitalize)
```
Comment (procaconsul) [inline lib/utils.ts:32]:
```suggestion
    .map(capitalize)
```

**Review by procaconsul — CHANGES_REQUESTED** [review]

File: lib/utils.ts (hunk 1 above)
Comment (zaki-amin) [reply lib/utils.ts]: Re-imported in 0ab3615d8bbd217d7b6e8a424da4612ad9896811

File: lib/utils.ts (hunk 2 above)
Comment (zaki-amin) [reply lib/utils.ts]: Adjusted in 0ab3615d8bbd217d7b6e8a424da4612ad9896811

**Review by procaconsul — APPROVED** [review]

**Pull request merged**